src/layouts/Pagination/styled.js

```diff
@@ -5,7 +5,7 @@ export const Box = styled.div`
   justify-content: center;
   flex-direction: row;
   align-items: center;
-  padding: 0px;
+  padding-bottom: 109px;
```

Comment:
Padding is reset in the mobile view because it is set padding to 0px on each side.

Reply:
Padding now is set to 31px to bottom, like in Figma project.